### PR TITLE
Set navbar fixed on scroll

### PR DESCRIPTION
**Pull request by joaojusto**
Why:

* We need to fix the navbar position when the page is scrolled

This change addresses the need by:

* Adding an `IntersectionObserver` to `<Navbar/>`;
* Toggling `.isFixed` class on `<Navbar/>` if the page was scrolled;
* Changing `<Navbar/>` styles accordingly;

### CODE REVIEW

**Comment by jfranciscosousa** [comment]
I think the animation should be just a tad bit longer.

**Comment by jfranciscosousa** [comment]
image
Also this is overflowing.

And the logo doesn't have text on the fixed version

**Comment by joaojusto** [comment]
@jfranciscosousa take a look now, I still need to do some cleaning on those hardcoded `width`, `height` values, but I'll do with after this is ready. Also, should I do mobile now, or are we doing it in another PR?

**Comment by jfranciscosousa** [comment]
You can do it now or on a later PR, totally your choice :ok_hand:

File: src/components/Navbar/index.module.css [hunk 1]
```diff
@@ -24,6 +35,10 @@
   position: absolute;
   top: $size-64;
   z-index: 1;
+
+  width: 140px;
```
Comment (jfranciscosousa) [inline src/components/Navbar/index.module.css]: I'm totally ok with fixed `px` values, just name them in vars so we can understand what they do.

**Review by jfranciscosousa — APPROVED** [review]

**Review by jfranciscosousa — APPROVED** [review]

**Comment by jfranciscosousa** [comment]
Can we make the navbar icon a bit bigger on tablets (big phones small desktops)?

Also, on mobile <700px the navbar height is still big, should be 93px ish

**Pull request merged**